AlfiyaSiddique/TastyTrails
# Profile Picture Functionality

## What does this PR do?



Fixes #237



## How should this be tested?




- Login into account and *click on edit button* provided besides profile picture
- Now click on any picture to *upload* from the file explorer that opened
- It will *update immediately on client* side and also *Base64* of the image will be *stored in Database*
- Now ***check Database*** (for Base64) and also try ***refreshing the page*** it should display correct image

https://github.com/user-attachments/assets/26572129-95ad-4c25-85e1-b1c1ca639d19



## Checklist



### Required

- [x] Removed all `console.logs` (**Except `console.logs`  for *errors***)
